.gitignore

```diff
@@ -52,3 +52,5 @@ app.*.map.json
 
 # test coverage file
 /coverage/lcov.info
+ios/Podfile
+ios/Podfile.lock
```

Comment:
To my knowledge, these should not be gitignored as they contain package version information for iOS or am I wrong?

Reply:
You are right, we ignored these files out of caution before researching their importance.

I'm removing the .gitignore entries here, but I'd like to see the podfiles added in a seperate branch

Reply:
See #114

Reply:
Perfect, all good 👍🏻